contribute.md

```diff
@@ -0,0 +1,26 @@
+# Contribute
+
+Please feel free to contribute to our project. Here are the instructions for anyone who wants to run this extension locally. 
+
+## Local Set Up
+
+If you desire to run this extension locally, here are the steps to have it run properly. 
+
+#### Specific instructions for setup:
+1. Clone the repo.
+2. Run npm install.
+3. Copy the .vscode.sample folder and rename it to .vscode.
+
+#### To verify that it's working:
+1. With VS code open, press F5.
+2. Another "Extension Development Host" VS code window should open.
+3. With a file open, right click.
+4. Click "Review Code File".
+5. An error message may pop up saying "APIKey not set". If not, then it should have opened a new tab showing your code and the suggested changes.
```

Comment:
Add a new step 3 that sets the API key in the configuration so this error doesn't happen. Then drop this item.